opts/envfile_test.go

```diff
@@ -84,15 +81,19 @@ func TestParseEnvFileBadlyFormattedFile(t *testing.T) {
     f   =quux
```

Comment:
@vdemeester any chance we can also test for other bad env names, like ones that start with a digit, and ones that have non-alphanumeric(_) chars in them (aside from spaces) - like "@" ?